src/pages/ae-waiting-time.js

```diff
@@ -15,6 +15,35 @@ const AECard = styled(Box)`
   align-items: center;
 `
 
+const HourLabel = styled(Typography)`
+  color: ${props => props.theme.palette.trafficLight[props.color]};
+`
+
+function WaitingTime(props) {
+  const { t } = useTranslation()
```

Comment:
please pass `t` from the react component instead of useTranslation here
or put this whole method inside the functional component

Reply:
`eslint` will throw error here or not?

Reply:
No eslint error

Reply:
The only warning is I shouldn't pass 'red' /'orange' as the props.color, but that's from console